src/sparseml/onnx/benchmark/info.py

```diff
@@ -311,101 +308,6 @@ def load_model(model: Any, **kwargs) -> ModelProto:
     return onnx.load(model)
 
 
-def load_data(
```

Comment:
we should ideally be able to keep some of the dataloading functionality in sparsezoo. were the call sites of this function updated? what functionality in sparseml was lost?

Reply:
unresolving - not responded

Reply:
added back in